config/config.go

```diff
@@ -213,6 +214,14 @@ func get(args []string, l *logrus.Entry) (cfg *Config, configFileF string, err e
 				l.Infof("Updating PMM Server address from %q to %q.", host, cfg.Server.Address)
 			}
 		}
+
+		// enabled cross-component PMM_DEBUG and PMM_TRACE take priority
+		if b, _ := strconv.ParseBool(os.Getenv("PMM_DEBUG")); b {
+			cfg.Debug = true
+		}
+		if b, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); b {
+			cfg.Trace = true
+		}
```

Comment:
Can be easier:
```suggestion
		cfg.Debug, _ = strconv.ParseBool(os.Getenv("PMM_DEBUG"))
		cfg.Trace, _ = strconv.ParseBool(os.Getenv("PMM_TRACE"))
```

Reply:
Hm, or can't

Reply:
Yeah, it will disable debug logging if PMM_DEBUG is not set, but PMM_AGENT_DEBUG is set to 1.